src/main/java/com/stripe/model/EventDataClassLookup.java

```diff
@@ -61,7 +60,6 @@
     classLookup.put("issuing.authorization", Authorization.class);
     classLookup.put("issuing.card", com.stripe.model.issuing.Card.class);
     classLookup.put("issuing.card_details", CardDetails.class);
-    classLookup.put("issuing.card_pin", CardPin.class);
```

Comment:
why did we remove this?

Reply:
Ah autogen had `CardPin` class before, and I confirmed with the issuing team that it is not for our client-libs but the mobiles.